recipes/traitlets/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "4.2.1" %}
+
+package:
+  name: traitlets
+  version: {{ version }}
+
+source:
+  fn: traitlets-{{ version }}.tar.gz
+  url: https://github.com/ipython/traitlets/archive/{{ version }}.tar.gz
+  md5: fc7f46a76b99ebc5068f99033d268dcf
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - ipython_genutils
+    - decorator
+    # enum34 may be of interest here also, but a test would also
+    # need to be added to the recipe. 
```

Comment:
> Personally, I would be happy to see the optional dependency with or without the test

That's like adding a function to a piece of software and having no idea if it is working or not - it makes me extremely nervous, especially given the simplicity of adding a test in this case.

In terms of running the test suite, if it takes <1 min to run I'm 👍 with it, otherwise the testing should be taking place in the source repository IMHO. We can't afford to eat up too many cycles with unittesting of packages in conda-forge.